packages/terra-clinical-action-header/src/ActionHeader.jsx

```diff
@@ -0,0 +1,177 @@
+   **/
+  onNext: PropTypes.func,
+
+  /**
+   * Callback function for when the previous button is clicked. The previous-next button group will not display if neither this or onNext are set.
+   **/
+  onPrevious: PropTypes.func,
+
+  /**
+   * Child element to be displayed on the right end of the header.
+   **/
+  children: PropTypes.element,
+};
+
+const defaultProps = {
+  title: '',
+  onClose: null,
+  onBack: null,
+  onExpand: null,
+  onMinimize: null,
+  onNext: null,
+  onPrevious: null,
+  children: null,
+};
+
+const ActionHeader = ({
+  title,
+  onClose,
+  onBack,
+  onExpand,
+  onMinimize,
+  onPrevious,
+  onNext,
+  children,
+  ...customProps
+}) => {
+  const attributes = Object.assign({}, customProps);
+  const closeButton = onClose ? <Button icon={<IconClose />} onClick={onClose} /> : null;
+  const backButton = onBack ? <Button icon={<IconLeft />} onClick={onBack} /> : null;
+
+  let closeButtonSmall;
+  let backButtonSmall;
+  if (onClose && !onBack) {
+    backButtonSmall = <Button icon={<IconLeft />} onClick={onClose} />;
+    closeButtonSmall = null;
+  } else {
+    closeButtonSmall = closeButton;
+    backButtonSmall = backButton;
+  }
+
+
+  let expandButton;
+  if (!backButton) {
+    if (onExpand) {
+      expandButton = <Button icon={<IconMaximize />} onClick={onExpand} />;
+    } else if (onMinimize) {
+      expandButton = <Button icon={<IconMinimize />} onClick={onMinimize} />;
+    }
+  }
+
+  let previousNextButtonGroup = null;
+  if (onPrevious || onNext) {
+    previousNextButtonGroup = (
+      <ButtonGroup>
+        <ButtonGroup.Button icon={<IconChevronUp />} onClick={onPrevious} key="ActionHeaderPrevious" />
+        <ButtonGroup.Button icon={<IconChevronDown />} onClick={onNext} key="ActionHeaderNext" />
+      </ButtonGroup>
+    );
+  }
+
+  const leftButtonsDefault = (
+    <div className="terraClinical-ActionHeader-leftButtons">
+      {backButton}
+      {expandButton}
+      {previousNextButtonGroup}
+    </div>
+  );
+
+  const rightButtonsDefault = (
+    <div className="terraClinical-ActionHeader-rightButtons">
+      {children}
+      {closeButton}
+    </div>
+  );
+
+  const leftButtonsSmall = (
+    <div className="terraClinical-ActionHeader-leftButtons">
+      {backButtonSmall}
+      {previousNextButtonGroup}
+    </div>
+  );
+
+  const rightButtonsSmall = (
+    <div className="terraClinical-ActionHeader-rightButtons">
+      {children}
+      {closeButtonSmall}
+    </div>
+  );
+
+  const actionHeader = (
+    <Header
+      {...attributes}
+      startContent={leftButtonsDefault}
+      title={title}
+      endContent={rightButtonsDefault}
+    />
+  );
+
+  const smallActionHeader = (
+    <Header
+      {...attributes}
+      startContent={leftButtonsSmall}
+      title={title}
+      endContent={rightButtonsSmall}
+    />
+  );
+
+  return (
+    <ResponsiveElement
+      responsiveTo="window"
+      defaultElement={smallActionHeader}
+      small={actionHeader}
```

Comment:
Refer here to see the responsiveElement api: http://engineering.cerner.com/terra-core/#/site/responsive-element
In this case the defaultElement/smallActionHeader will be displayed from 0 to the small breakpoint, at the small breakpoint and larger the actionHeader would be displayed.